githubapp/ldap.py

```diff
@@ -106,7 +106,9 @@ def get_group_members(self, group_name):
                                 else:
                                     email = None
                                 if "EMU_SHORTCODE" in os.environ:
-                                    username = username + "_" + os.environ["EMU_SHORTCODE"]
+                                    username = (
```

Comment:
confirming this is just formatting?  looks like it, but if I am missing something it may need to go in the other files that this bit was added to

Reply:
yes, that's just formatting from `black`

Reply:
the only material changes were made to `Pipfile` (removing the explicit version of `black`) and `Pipfile.lock` (re-locking all dependencies to the current LTS versions)